zio-http/src/main/scala/zhttp/http/Http.scala

```diff
@@ -644,8 +645,8 @@ object Http {
   /**
    * Creates an Http app from a resource path
    */
-  def fromResource(path: String): HttpApp[Any, Throwable] =
-    Http.getResourceAsFile(path) >>= { Http.fromFile(_) }
+  def fromResource(path: String): HttpApp[Blocking, Throwable] =
+    Http.getResource(path).flatMap(url => Http.fromFile(new File(url.getPath)))
```

Comment:
This change is not related to the fix of `Http.getResource`.
It just help to delay the creation of the `File` instance as much as possible.